components/CredentialField.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div v-if="show">
+    <q-item class="col q-py-none">
+      <q-item-section>
+        <q-item-label
+          v-if="title"
+          :class="titleClass"
+          :lines="lines">
+          {{title}}
+        </q-item-label>
+        <q-item-label
+          :class="valueClass"
+          :lines="lines">
+          {{value}}
+        </q-item-label>
+      </q-item-section>
+    </q-item>
+  </div>
+</template>
+
+<script setup>
+import {computed, defineProps} from 'vue';
```

Comment:
Missing copyright header here and anywhere else.

Reply:
4a1122145b9b4b1c9023ab51062930e14c2fe365